webpack/scenes/ContentViews/Details/Filters/CVModuleStreamFilterContent.js

```diff
@@ -209,6 +216,15 @@ const CVModuleStreamFilterContent = ({
                   }
                   />
                 </SplitItem>
+                <SplitItem>
+                  <Checkbox
+                    id="moduleStreamNoErrata"
+                    name="moduleStreamNoErrata"
+                    label={originalStreamsLabel}
```

Comment:
```suggestion
                   label=<p style={{marginTop:'4px'}}>{originalStreamsLabel}</p>
```

The checkboxes have a 4px margin added which doesn't apply to labels. This makes it better aligned on the page.

Reply:
updated!.